plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/visitors/UpdateScmUrlVisitor.java

```diff
@@ -0,0 +1,42 @@
+package io.jenkins.tools.pluginmodernizer.core.visitors;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.maven.MavenIsoVisitor;
+import org.openrewrite.xml.tree.Content;
+import org.openrewrite.xml.tree.Xml;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UpdateScmUrlVisitor extends MavenIsoVisitor<ExecutionContext> {
+
+    @Override
+    public Xml.Tag visitTag(Xml.Tag tag, ExecutionContext context) {
+        tag = super.visitTag(tag, context);
+        if ("scm".equals(tag.getName())) {
```

Comment:
Instead of using lot of level of conditional, we can return directly if the tag is not what we expect. 

This would improve readability of the code 

```java 
if(!"scm".equals(tag.getName()) {
   return tag;
}
```